package.json

```diff
@@ -1,8 +1,8 @@
 {
-  "name": "svelte-app",
-  "version": "1.0.0",
+  "name": "ion_core",
+  "version": "2.2.0",
   "scripts": {
-    "build": "rollup -c",
+    "build": "rollup -c && rollup -c rollup.config.addon.js --config-study-list-url=/public/locally-available-studies.json && web-ext build --overwrite-dest && mv web-ext-artifacts/*.zip web-ext-artifacts/ion_core.xpi",
```

Comment:
@rhelmer why are you changing this command?

Do we need to add another command to package a release? In the study template, for example, you recommended adding the [`package` command](https://github.com/mozilla-ion/ion-basic-study/blob/75249cd338da6b10cc7e38e7ff72d290323b18da/package.json#L7).

Reply:
There's a slack thread I started in #addons-pipeline about this, basically the `build` target is hardcoded in the taskcluster CI AFAIK :( Trying to figure this out though I am seeing some additional runs for some of our other targets so this needs investigation.

I *think* what the CI does is run `yarn build`, so we might just be seeing the effect of whatever that does. I don't think we should switch to `yarn` but we need to support it somewhat for now I guess.